src/redux/countyReducer.js

```diff
@@ -0,0 +1,50 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// Thunk makes API call, returns action of the type specified by the first parameter
+// In the slice, extraReducers define different actions based on the 3 possible states of the returned promise and execute reducers for them
+export const countyFetch = createAsyncThunk(
+  'counties/countyFetch',
+  //payload creator
+  async () => {
+    const url = "http://localhost:3001/counties"
+    try {
+      const res = await axios.get(url);
+      return res.data; 
+    } catch(err) {
+        console.error("There was a problem fetching county data" + err);
+        return [];
+    }  
+  }
+);
+
+// Then, handle actions in your reducers:
+const countySlice = createSlice({
+  name: 'counties',
+  initialState: { counties: {}, status: 'idle', error: null },
+  reducers: {
```

Comment:
Doublechecking this empty reducers argument object; if there are no actions needed, should this be included? Or a comment to the effect that there are no action types?